src/bin/sol-fbp-generator/main.c

```diff
@@ -1886,6 +1886,9 @@ print_usage(const char *program)
         "    -j  When resolving types, use the passed DESC files. If DESC is\n"
         "        a directory then all the .json files in the directory will be used.\n"
         "        Multiple -j can be passed.\n"
+        "        Note that Soletta install dir will also be looked, as well as any\n"
```

Comment:
+1 to Gustavo suggestion. Just make sure that this is clear from the help ("if no env var is set, take the soletta installation").